src/components/backend-ai-error-log-list.ts

```diff
@@ -147,17 +146,17 @@ export default class BackendAiErrorLogList extends BackendAIPage {
    * Refresh log data.
    */
   _refreshLogData() {
-    this.list_condition = 'loading';
-    this.list_status.show();
+    this.listCondition = 'loading';
+    this._listStatus?.show();
     this._updatePageItemSize();
     this.logs = JSON.parse(localStorage.getItem('backendaiwebui.logs') || '{}');
     this._totalLogCount = this.logs.length > 0 ? this.logs.length : 1;
     this._updateItemsFromPage(1);
     this._grid.clearCache();
     if (this.logs.length == 0) {
-      this.list_condition = 'no-data';
+      this.listCondition = 'no-data';
```

Comment:
Just a quick suggestion, how about adding a type on `listCondition` on every component that uses that property, if there are only two types used in there? If does, It will check whether the assigning value is valid or not in compile time(inside IDE).
Here's a good example that uses type definition on string-based values in `backend-ai-session-list` component.
https://github.com/lablup/backend.ai-webui/blob/63a1b1d0891933e2cddcc2aa38972c0185a12d47/src/components/backend-ai-session-list.ts#L73-L85

Reply:
Good suggestion! I'll apply right away thanks :)

Reply:
Let's apply this to a separate PR.

Reply:
I'll handle it another PR 😉